sunpy/net/dataretriever/client.py

```diff
@@ -225,7 +107,34 @@ def _can_handle_query(cls, *query):
         `sunpy.net.fido_factory.UnifiedDownloaderFactory`
         class uses to dispatch queries to this Client.
         """
-        raise NotImplementedError
+        from sunpy.net import attrs as a
+
+        required = {a.Time, a.Instrument}
+        adict = cls.register_values()
+        optional = {i for i in adict.keys()}
+        if not cls.check_attr_types_in_query(query, required, optional):
+            return False
+        for x in query:
+            if isinstance(x, a.Instrument) and x.value.lower() == adict[a.Instrument][0][0].lower():
+                return True
+        return False
+
+    def post_hook(self, exdict, matchdict):
+        """
+        """
+        map_ = OrderedDict()
+        start = parse_time("{}/{}/{}".format(exdict['year'], exdict['month'], exdict['day']))
+        end = start + TimeDelta(1 * u.day - 1 * u.millisecond)
+        map_['Start Time'] = start.strftime(TIME_FORMAT)
```

Comment:
Why are we converting these to strings to put them in the response, can't we just pass astropy time objects to Table?

Reply:
Done